PlayCover/Utils/Cacher.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Cacher.swift
+//  PlayCover
+//
+//  Created by Amir Mohammadi on 10/2/1401 AP.
+//
+
+import Foundation
+import AppKit
+import DataCache
+
+class Cacher {
+
+    let cache = DataCache.instance
+    /// We can create a custom cache like this (default values are as the same as below):
+    /// `let cache = DataCache(name: "PlayCoverCache")`
+    /// `cache.maxDiskCacheSize = 100*1024*1024`      // 100 MB
+    /// `cache.maxCachePeriodInSecond = 7*86400`      // 1 week
+    /// More details: https://github.com/huynguyencong/DataCache/blob/master/README.md
+
+    func resolveITunesData(_ link: String) async {
+        if let refreshedITunesData = await getITunesData(link) {
+            do {
+                try cache.write(codable: refreshedITunesData, forKey: link)
+            } catch {
+                print("Write error \(error.localizedDescription)")
+            }
```

Comment:
If error is not handled and just ignored then do-catch is excessive 
`try? cache.write(codable: refreshedITunesData, forKey: link)` is enough